assets/stylesheets/bootstrap/_theme.scss

```diff
@@ -134,6 +136,7 @@
 // Default navbar
 .navbar-default {
   @include gradient-vertical($start-color: lighten($navbar-default-bg, 10%), $end-color: $navbar-default-bg);
+  @include reset-filter; // Remove gradient in IE<10 to fix bug where dropdowns don't get triggered
```

Comment:
I'm not sure what reset-filter is doing here, are you sure we need this? At the very least remove the mention of IE9 if you're 100% sure that this is somehow required (doubtful).